index.html

```diff
@@ -333,12 +333,240 @@
+          may not support continuous availability monitoring; for example,
+          because of platform or power consumption restrictions. In this case
+          the <a>Promise</a> returned by <code><a for="RemotePlayback">getAvailability</a>()</code>
+          MUST be <a data-lt="reject">rejected</a> and the algorithm to <a>monitor
+          the list of available remote playback devices</a> will only run as part of the
+          <a>initiate remote playback</a> algorithm.
+        </p>
+        <p>
+          When there are no live <a>RemotePlaybackAvailability</a> objects
+          (that is, the <a>set of availability objects</a> is empty), user
+          agents SHOULD NOT <a>monitor the list of available remote playback devices</a> to satisfy the
+          <a>power saving non-functional requirement</a>. To further save power,
+          the <a>user agent</a> MAY also keep track of whether the page
+          holding a <a>RemotePlaybackAvailability</a> object is in the
+          foreground. Using this information, implementation specific
+          discovery of <a>remote playback devices</a> can be resumed or
+          suspended.
+        </p>
+        <p>
+          Some <a>remote playback devices</a> may only be able to display a
+          subset of Web content because of functional, security or hardware
+          limitations. Examples are set-top boxes, smart TVs or networked
+          speakers capable of rendering only certain formats of video and/or audio.
+          We say that such a device is a <dfn>compatible remote playback device</dfn> for a
+          <a>media element source</a> if the <a>user agent</a> can
+          reasonably guarantee that the remote playback of the media speficied by the source
+          on that device will succeed.
+        </p>
+      </section>
+      <section>
+        <h4>
+          Getting the <a>remote playback devices</a> availability information
+        </h4>
+        <p>
+          When the <code><dfn for="RemotePlayback">getAvailability</dfn>()</code> method is
+          called, the user agent MUST run the following steps:
+        </p>
+        <dl>
+          <dt>
+            Input
+          </dt>
+          <dd>
+            <var>remote media source</var>, the <a>media element source</a> preferred for remote playback
+          </dd>
+          <dt>
+            Output
+          </dt>
+          <dd>
+            <var>P</var>, a <a>Promise</a>
+          </dd>
+        </dl>
+        <ol>
+          <li>Let <var>P</var> be a new <a>Promise</a>.
+          </li>
+          <li>Return <var>P</var>, but continue running these steps <a>in
+          parallel</a>.
+          </li>
+          <li>If the user agent is unable to <a>monitor the list of available
+          remote playback devices</a> for the entire lifetime of the browsing context
+          (for instance, because the user has disabled this feature), then:
```

Comment:
Could you elaborate a little more? Do you think this passage is excessive as part of the algorithm? Or do you think we shouldn't reject the promise in this case?